CHANGELOG.rst

```diff
@@ -12,18 +12,28 @@ Internal Changes
 
 * **Gateway**: Add endpoint for refreshing expiring Gitlab tokens
 * **Sessions**: Check LFS size and available disk space before cloning
+* **Knowledge Graph**: Improve access tokens management
+* **Knowledge Graph**: Add migration importing Composite Plans into the Triples Store
+* **Knowledge Graph**: Prevent from data corruption caused by duplicate Project creation dates
 
 **Bug fixes**
 
 * **Sessions**: Handle expiring Gitlab tokens
 * **Gateway**: Do not remove Redis clients on logout
 * **Gateway**: Address security vulnerabilities
+* **Knowledge Graph**: Fix vulnerabilities in Graph Services' docker images
+* **Knowledge Graph**: Handle cases when Plan Invalidation Time is wrong
+* **Knowledge Graph**: Made Cross-Entity Search API to sort in a case-insensitive way
```

Comment:
Added case-insensitive sorting to Cross-Entity Search API

Reply:
Wouldn't that suggest that case-insensitive sorting is an option and you can still be able to sort case-sensitive? Actually, case-insensitive sorting is a replacement for case-sensitive.

Reply:
Ok - how about: `Modified Cross Entity search API to perform case-insensitive sorting`